src/js/entities-service/entities/clinicians.js

```diff
@@ -49,9 +50,15 @@ const _Model = BaseModel.extend({
       || this.get('access') === 'admin';
   },
   saveRole(role) {
+    const url = `/api/clinicians/${ this.id }/relationships/role`;
     role = this.toRelation(role);
-    return this.save({ _role: role.data }, {
-      relationships: { role },
+
+    this.set({ _role: role.data.id });
```

Comment:
can we do this set prior to the `toRelation` so it can just be `role.id` .. maybe the `toRelation` can be inside the `stringify` so we're not modifying `role`